VERSION

```diff
@@ -1 +1 @@
-2.0.0-minerva.0.1.0
+2.0.0-rc.1
```

Comment:
```suggestion
2.0.0-rc1
```

Reply:
My definition of pretty is actually the former version.